### ISSUE TITLE
memory_usage is [] on A6000s

### ISSUE THREAD

**Issue by jturner116** [triton-inference-server/server#6009]
**Description**
The new `memory_usage`  metric in statistics extension is returning as an empty list

**Triton Information**
2.35.0

Are you using the Triton container or did you build it yourself?
Triton Container

**To Reproduce**
Steps to reproduce the behavior.
Running on A6000s with no MIG enabled (not even sure if MIG is possible on A6000s)

server cli args 
```tritonserver --model-repository=/models --strict-model-config=false --backend-config=onnxruntime,default-max-batch-size=32,triton-backend-memory-tracker=true```

Model config
```
name: "snvin_ner"
backend: "onnxruntime"
max_batch_size: 8
input [
    {
        name: 'attention_mask'
        data_type: TYPE_INT64
        dims: [ -1]
    },
    {
        name: 'input_ids'
        data_type: TYPE_INT64
        dims: [ -1]
    }
]
output [
    {
        name: 'logits'
        data_type: TYPE_FP32
        dims: [-1,8]
    }
]

instance_group [
  {
    count: 1
    kind: KIND_GPU
    gpus: [ 1 ]
  }
]
dynamic_batching 
  {
    preferred_batch_size: 2
  }
parameters { key: "gpu_mem_limit" value: { string_value: "650000000" } }
parameters { key: "arena_extend_strategy" value: { string_value: "1" } }
```

**Expected behavior**
I don't believe any of my current configuration parameters would affect the memory usage stat. I apologize if I have missed an incompatibility somewhere. 

Here is current output
```
{"model_stats":[{"name":"snvin_ner","version":"2","last_inference":1688136584412,"inference_count":100,"execution_count":10,"inference_stats":{"success":{"count":10,"ns":239197232},"fail":{"count":0,"ns":0},"queue":{"count":10,"ns":2044037},"compute_input":{"count":10,"ns":1332935},"compute_infer":{"count":10,"ns":233408267},"compute_output":{"count":10,"ns":597639},"cache_hit":{"count":0,"ns":0},"cache_miss":{"count":0,"ns":0}},"batch_stats":[{"batch_size":4,"compute_input":{"count":5,"ns":545136},"compute_infer":{"count":5,"ns":52170915},"compute_output":{"count":5,"ns":180297}},{"batch_size":16,"compute_input":{"count":5,"ns":787799},"compute_infer":{"count":5,"ns":181237352},"compute_output":{"count":5,"ns":417342}}],"memory_usage":[]}]}
```

Thanks!
Jonah

**Comment by oandreeva-nv** [comment]
@jturner116 Thank you for letting us know about this issue. I will create a ticket for us to investigate this issue. (CC @GuanLuo )

**Comment by GuanLuo** [comment]
Does the server log contain something related to not being able to capture memory usage for the model?

**Comment by jturner116** [comment]
@GuanLuo I feel silly for not having checked, I will take a peek at the logs when I get access to the server again on Monday. Thanks for your help

**Comment by jturner116** [comment]
Here are some relevant portions of the logs
```
I0703 12:09:42.103002 1 onnxruntime.cc:2530] TRITONBACKEND_Initialize: onnxruntime
I0703 12:09:42.103028 1 onnxruntime.cc:2540] Triton TRITONBACKEND API version: 1.13
I0703 12:09:42.103033 1 onnxruntime.cc:2546] 'onnxruntime' TRITONBACKEND API version: 1.13
I0703 12:09:42.103038 1 onnxruntime.cc:2576] backend configuration:
{"cmdline":{"auto-complete-config":"true","backend-directory":"/opt/tritonserver/backends","default-max-batch-size":"32,triton-backend-memory-tracker=true","min-compute-capability":"6.000000"}}
```
```
+-------------+-----------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Backend     | Path                                                            | Config                                                                                                                                                                                            |
+-------------+-----------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| pytorch     | /opt/tritonserver/backends/pytorch/libtriton_pytorch.so         | {}                                                                                                                                                                                                |
| onnxruntime | /opt/tritonserver/backends/onnxruntime/libtriton_onnxruntime.so | {"cmdline":{"auto-complete-config":"true","backend-directory":"/opt/tritonserver/backends","default-max-batch-size":"32,triton-backend-memory-tracker=true","min-compute-capability":"6.000000"}} |
+-------------+-----------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
```
--------------------------------------------------------------+
| Option                           | Value                                                                                                                                                                                                           |
+----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| server_id                        | triton                                                                                                                                                                                                          |
| server_version                   | 2.35.0                                                                                                                                                                                                          |
| server_extensions                | classification sequence model_repository model_repository(unload_dependents) schedule_policy model_configuration system_shared_memory cuda_shared_memory binary_tensor_data parameters statistics trace logging |
| model_repository_path[0]         | /models                                                                                                                                                                                                         |
| model_control_mode               | MODE_NONE                                                                                                                                                                                                       |
| strict_model_config              | 0                                                                                                                                                                                                               |
| rate_limit                       | OFF                                                                                                                                                                                                             |
| pinned_memory_pool_byte_size     | 268435456                                                                                                                                                                                                       |
| cuda_memory_pool_byte_size{0}    | 67108864                                                                                                                                                                                                        |
| cuda_memory_pool_byte_size{1}    | 67108864                                                                                                                                                                                                        |
| min_supported_compute_capability | 6.0                                                                                                                                                                                                             |
| strict_readiness                 | 1                                                                                                                                                                                                               |
| exit_timeout                     | 30                                                                                                                                                                                                              |
| cache_enabled                    | 0                                                                                                                                                                                                               |
```
@GuanLuo Is there anything else I am missing? Thanks for your help!!

**Comment by GuanLuo** [comment]
Hi @jishminor , sorry for the late reply. I have figured out that the issue is from the cmdline option for setting backend configuration: to specify multiple configuration pairs, they need to be specified separately (see below). Can you give it a try to see if the memory usage will be reported?
before:
```
--backend-config=onnxruntime,default-max-batch-size=32,triton-backend-memory-tracker=true
```
after:
```
--backend-config=onnxruntime,default-max-batch-size=32 --backend-config=triton-backend-memory-tracker=true
```

**Comment by GuanLuo** [comment]
closing due to inactivity, please reopen with additional information if the above suggestion doesn't address the issue.
